src/format.rs

```diff
@@ -206,5 +206,7 @@ pub enum FormatLayout {
     I32 = __gl::INT,
     F16 = __gl::HALF_FLOAT,
     F32 = __gl::FLOAT,
+    U24U8 = __gl::UNSIGNED_INT_24_8,
```

Comment:
I added these names because they are the only valid types for using `read_pixels` with `DepthStencil`. I'm open to better naming suggestions.